pywwt/annotation.py

```diff
@@ -62,6 +63,26 @@ def _on_trait_change(self, changed):
                                   setting=wwt_name,
                                   value=changed['new'])
 
+    def _serialize_state(self):
+        state = {'shape': self.shape,
+                 'id': self.id,
+                 'settings': []}
```

Comment:
Here and for the layers, the settings are stored as a list of dicts, each of which is currently a name/value pair. Should we instead store the settings as a dictionary instead?

```
"settings": {
  "opacity": 1.0
}
```

One obvious thing to think about is whether a setting will ever need to contain a complex value, but this can be accomplished by sometimes using a sub-dictionary:

```
"settings": {
  "opacity": 1.0,
  "color": { "r": 255, "g": 100, "b": 123, "my-metadata": "added on 2019 June 25" }
}
```

This change would affect the data storage format so I'd like to make sooner rather than alter if that's the path we go down. It's not a huge difference, but if you imagine altering the JSON structures with code, I think it will make life substantially easier to have a dict interface rather than iterating through lists (as in the test module right now).